### PR TITLE
[17.0][MIG] crm_lead_currency: Migration to 17.0 

### PR DESCRIPTION
**Pull request by BernatObrador**
Module migrated to version 17.0

cc https://github.com/APSL 158348

@miquelalzanillas @lbarry-apsl @mpascuall @peluko00 @javierobcn @ppyczko please review

### CODE REVIEW

**Review by mpascuall — APPROVED** [review]
LGTM, tested in runboat

**Review by lbarry-apsl — APPROVED** [review]
Tested on runboat. LGTM

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by peluko00 — COMMENTED** [review]
LGTM, tested in runboat

**Review by peluko00 — APPROVED** [review]
LGTM, tested in runboat

File: crm_lead_currency/views/crm_lead_views.xml [hunk 1]
```diff
@@ -12,15 +12,10 @@
...
-                    name="amount_customer_currency"
-                    attrs="{'invisible': [('is_same_currency', '=', True)]}"
-                />
+                <field name="amount_customer_currency" invisible="is_same_currency" />
             </xpath>
             <field name="expected_revenue" position="attributes">
-                <attribute name="attrs">
-                    {'readonly': [('is_same_currency', '=', False)]}
-                </attribute>
+                <attribute name="readonly" add="(not is_same_currency)" />
```
Comment (pedrobaeza) [inline crm_lead_currency/views/crm_lead_views.xml]:
```suggestion
                <attribute name="readonly">not is_same_currency</attribute>
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: crm_lead_currency/views/crm_lead_views.xml (hunk 1 above)
Comment (BernatObrador) [reply crm_lead_currency/views/crm_lead_views.xml]: Done, ready to review

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**